keychain/models/keychain.py

```diff
@@ -187,11 +193,14 @@ def _get_keys(envs):
                 if key and len(key) > 0  # remove False values
             ]
 
-        if force_env:
-            envs = [force_env]
+        if serv_config and serv_config.has_option('keychain', 'key'):
+            keys = [Fernet(serv_config.get('keychain', 'key'))]
```

Comment:
why not using MultiFernet like in the last instruction of this method ?

Reply:
These `keys` will be passed to that MultiFernet.

Reply:
Yes, I read to quickly :raised_hand_with_fingers_splayed:  :hammer: